test/libsolidity/semanticTests/abiEncoderV1/bool_out_of_bounds.sol

```diff
@@ -3,9 +3,10 @@ contract C {
 	function f(bool b) public pure returns (bool) { return b; }
 }
 // ====
+// compileViaYul: false
```

Comment:
Please revert these, there is no `abi.encodeCall` in these tests, so they should not change.

Reply:
Reverted the changes in the latest push.